content/en/metrics/faq/updating-to-distribution-metrics-faq.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Updating to Distribution Metrics FAQ
+kind: faq
+is_beta: true
+further_reading:
+- link: "/logs/faq/how-to-send-logs-to-datadog-via-external-log-shippers/"
+  tag: "FAQ"
+  text: "How to Send Logs to Datadog via External Log Shippers?"
+---
+
+## Why has my Distribution Metrics page been deprecated?
+
+Datadog has a new pipeline for calculating globally accurate percentiles for your distribution metrics - this unlocks more query functionality and an easier configuration workflow for distributions.
+
+## How does this new pipeline for Distributions affect me? 
+
+- Unlocks new query functionalities:
+  - Accurate percentiles in time for any historical timeframe (i.e., the past day, week, month) of their choosing.
```

Comment:
- Accurate percentiles in time for any historical timeframe (i.e., the past day, week, month)